.travis.yml

```diff
@@ -143,6 +143,8 @@ install:
 - conda info -a
 - conda create -q -n p4env python=$PYTHON_VER numpy cmake ci-psi4-lt psi4-lt-mp gau2grid dftd3 gcp snsmp2 scipy -c psi4/label/dev -c psi4
 - source activate p4env
+- conda install hungarian networkx -c psi4
+- conda install deepdiff -c conda-forge
```

Comment:
Are these actively needed by Psi4? Seems like quite a few new depends.

Reply:
Yeah, pains me a bit, too. They're all there for the comparison of Mol to its dict repr. They're definitely needed for a few tests. Right now they're needed for everything b/c I put in an extra every-time check to satisfy myself it was working in volume. Been ok for a couple months now, so possibly removeable.

The deepdiff doesn't bother me - its conda and pip-able. but the hungarian and networkx are rather overkill for everyday dependencies. Will have to consider them further.